dist/profile/manifests/jiraveg.pp

```diff
@@ -0,0 +1,32 @@
+# profile for jira
+class profile::jiraveg (
+  $jira_javahome,
+  $jira_version
+){
+  $pkgs = ['openjdk-7-jre-headless', 'facter']
```

Comment:
There's a puppetlabs/java module which is probably a better idea to include instead of declaring the openjdk dependency directly